### ISSUE TITLE
INP not showing on self-hosted

### ISSUE THREAD

**Issue by amenk** [getsentry/self-hosted#2970]
I am currently testing the new INP integration locally.

Versions:

* Version 122.0.6261.128 (Official Build) (64-bit)
* Sentry Browser Integration https://browser.sentry-cdn.com/7.110.0/bundle.tracing.min.js
* Sentry 24.3.0 self hosted, with the default features from the config example

Tracing sample rate is set to 1

I used

```
    Sentry.browserTracingIntegration({
        enableInp: true,
    }),
```

The web vitals view looks like this

![image](https://github.com/getsentry/sentry-javascript/assets/1087128/e77a0546-3192-42f2-b232-bc7f56fe4d36)

I did not deploy the enableInp yet to production, maybe it would work there with a greater user base?

I am mostly interested in the INP element - how would this show up? Can someone post an example how it should look like when it is working?

When exactly is the INP transmitted? I saw in https://github.com/getsentry/sentry-javascript/discussions/7765 that some caching might be in place?

_Originally posted by @amenk in https://github.com/getsentry/sentry-javascript/discussions/11591_

**Comment by aldy505** [comment]
Can you try to add `"organizations:performance-vitals-inp"` to the feature flag on to the list of feature flags that's defined here? https://github.com/getsentry/self-hosted/blob/d59c0aa57c1432af9c5000007c17a87c07935a55/sentry/sentry.conf.example.py#L266-L312

Then re-run `./install.sh` script.

**Comment by amenk** [comment]
@aldy505 Added ist and re-ran install.sh - still "no data" after browsing a bit on the page.
How to sort out if the browser is sending INP? I have the debug javascript package already included.

```
<script src="https://browser.sentry-cdn.com/7.110.0/debug.min.js" crossorigin="anonymous"></script>
```

**Comment by aldy505** [comment]
Okay I've tested it on my instance. All you need to do:

1. Add another feature flag on `sentry/sentry.conf.py`
    ```py
    SENTRY_FEATURES.update(
      # ...
      "organizations:performance-vitals-inp",
      # ...
    )
    ```
2. Re-run `sudo ./install.sh && sudo docker compose up -d`
3. I'm using this bundle: `https://browser.sentry-cdn.com/7.111.0/bundle.tracing.replay.min.js`
4. On `Sentry.init`, add `enableInp: true`
    ```js
    Sentry.init({
      // ...
      integrations: [
        Sentry.browserTracingIntegration({
          enableInp: true
        })
      ]
    })
    ```

Event should be coming after sometime, it's not gonna be instant.

@amenk Let me know if that's working for you

**Comment by zKoz210** [comment]
@aldy505 thanks, it’s work!

**Comment by aldy505** [comment]
Seeing the sentry repo, the `organizations:performance-vitals-inp` shouldn't be guarded anymore since... I don't know what version, probably 24.4.2? I'm going to confirm this later (by removing the feature flag on my instance) in a few hours and close this issue if I can see it's working.

**Comment by hubertdeng123** [comment]
Yep, this should now be fixed on 24.4.2+
